test/unit/math/prim/fun/symmetrize_from_lower_tri_test.cpp

```diff
@@ -22,6 +22,6 @@ TEST(Symmetrize, Value) {
   Matrix<double, Dynamic, Dynamic> m(4, 4);
   for (int i = 0; i < 4; ++i)
     for (int j = 0; j < 4; ++j)
-      m(i, j) = (i + 1) * (j + 1);
+      m(i, j) = (i + 2) * (j + 1);
```

Comment:
Ooooh, we should be checking that this function gives the right result.

So build the result:
```
Eigen::MatrixXd m_ref(4, 4);
m_ref << a, b, c, d,
    b, e, f, ...;
EXPECT_MATRIX_EQ(m_sym_ref, symmetrize_from_lower_tri(m));
```

Reply:
Thanks Ben, I'll fix these issues.